invenio.cfg

```diff
@@ -434,12 +434,15 @@ RDM_RECORDS_IDENTIFIERS_SCHEMES = {**RDM_RECORDS_IDENTIFIERS_SCHEMES,
                                             "validator": schemes.is_indico,
                                             "datacite": "INDICO"
                                         },
-                                               },
                                       "hdl": {
                                           "label": _("Handle"),
                                           "validator": schemes.is_handle,
-                                          "datacite": "HANDLE",
-                                      },
+                                          "datacite": "HANDLE"},
+                                      "cdsrdm": {
```

Comment:
it should be `cds`, not `cdsrdm`. See https://github.com/CERNDocumentServer/cds-rdm/issues/496